src/main/java/com/chargerrobotics/subsystems/ShooterHoodSubsystem.java

```diff
@@ -57,6 +57,43 @@ public boolean isLimitSwitchTriggered() {
     public double getHoodPosition() {
         return (double)shooterHood.getSensorCollection().getQuadraturePosition();
     }
+    
+    public double getHoodAngle()  {
+        LimelightSubsystem limelight = LimelightSubsystem.getInstance();
+        double d = limelight.distance();
+        //double d needs to be the information coming from the limelight, the length reported from the limelight
+        double h = (Constants.targetHeight - Constants.cameraHeight);
+        //conversion from inches to meters    
+        double theta = Math.atan(2*h/d);
+        //calculations for theta of equatio
+    
+        return Math.toDegrees(theta); 
+    }
+
+    public double getAngleByZone() {
+        LimelightSubsystem limelight = LimelightSubsystem.getInstance();
+        //double d = limelight.distance();
+        double d = 50.0; //sample distance since limelight wasn't working
```

Comment:
Since this value was only used for testing, we shouldn't leave it in the code since it could just cause lots of confusion.